.github/SECURITY.md

```diff
@@ -1,61 +1,50 @@
- .. Licensed to the Apache Software Foundation (ASF) under one
-    or more contributor license agreements.  See the NOTICE file
-    distributed with this work for additional information
-    regarding copyright ownership.  The ASF licenses this file
-    to you under the Apache License, Version 2.0 (the
-    "License"); you may not use this file except in compliance
-    with the License.  You may obtain a copy of the License at
```

Comment:
Is the Apache License preamble not needed?

Reply:
Good point. It needs to be added. For some reason it's not been added automatically (I tend not to think about those because they are added automatically). I will fix both. It's not **technically** needed though. The preamble is needed when we distribute sources. This file is not part of the "-sources.tgz", and this is where licences are needed.

But it does not hurt to add it.

Reply:
added here.  Also https://github.com/apache/airflow/pull/32276 will add them automaticaly in .github folder as well.